_dev/coding-style.md

```diff
@@ -6,8 +6,65 @@ description: The preferred style for code committed to NeoMutt
 
 # Coding Style
 
-Every project has its own coding style, this one is no exception.
-However, I haven't written down the rules.  For now, ...
+## at the moment (pre-fork)
 
-- Trailing whitespace is **evil**
+Generally speaking, we don't want to diverge much from upstream mutt. We're
+still merging patches in - and if we didn't follow upstream's coding style,
+merging would be much harder.
```

Comment:
As long as we're not modifying the existing code, it does not matter much what style we apply for new code. It would only matter for the rare occasions we're pushing back code to upstream. But I believe this is a rare enough case that we'd better be modifying the patch to match the coding style of upstream when it happens, than keep rules enforced upstream.